quickstarts/microsoft.hybridcompute/machine-windows-configuration/README.md

```diff
@@ -0,0 +1,89 @@
+
+General information about how configurations are assigned to machines in Azure is available in documentation.
+
+[Understand the guest configuration feature of Azure Policy](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/guest-configuration)
+
+A detailed how to document about assigning configurations to machines, and how to customize configurations from ARM,
+is also available.
+
+## Required prerequisites
+
+Before testing this quickstart template, you must create a Windows machine
+(virtual or physical) outside of Azure and install the Arc agent
+to project it into Azure.
+
+The simplest approach to building this prereq is to follow the docs page
+for Azure Arc-enabled servers.
+
+[Enable Arc-enabled servers agent](https://docs.microsoft.com/en-us/azure/azure-arc/servers/learn/quick-enable-hybrid-vm)
+
+After you have an Arc-enabled machine ready, you can deploy this sample
+in to the same resource group and use the "machineName" parameter to specify
+the machine where the configuration will be applied.
+
+
+---
+
+**Common administration ports to log in to the VM directly, are not opened in this template.**
+After deploying this machine, it is expected that you will
+[deploy applications](https://docs.microsoft.com/azure/devops/pipelines/release/deployment-groups/deploying-azure-vms-deployment-groups)
+using a service such as Azure DevOps.
+
+- [Provision agents for deployment groups](https://docs.microsoft.com/azure/devops/pipelines/release/deployment-groups/howto-provision-deployment-group-agents)
+
+To provide ongoing operations for the machine, it is expected you will use the available
+Azure management services. You can also view the
+[Cloud Adoption Framework](https://docs.microsoft.com/azure/cloud-adoption-framework/manage/azure-server-management/)
+for more information about best practices.
+
+- [Azure Automanage](https://docs.microsoft.com/azure/automanage/automanage-virtual-machines)
+- [Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/)
+- [Azure Update Management](https://docs.microsoft.com/azure/automation/update-management/overview)
+- [Azure Automation inventory feature](https://docs.microsoft.com/azure/automation/change-tracking/manage-inventory-vms)
+- [Azure Policy's guest configuration feature](https://docs.microsoft.com/azure/governance/policy/concepts/guest-configuration)
+- [Azure Backup](https://docs.microsoft.com/azure/backup/)
+- [Azure Custom Script extension for Windows](https://docs.microsoft.com/azure/virtual-machines/extensions/custom-script-windows)
+- [Azure Run Commands for Windows](https://docs.microsoft.com/azure/virtual-machines/windows/run-command)
+
+If you would prefer to open common ports, modify rules in the network security group
+associated with the network adapter for the machine.
+
+- [Network security groups](https://docs.microsoft.com/azure/virtual-network/network-security-groups-overview)
```

Comment:
This section is irrelevant for on-prem machines.

Reply:
removed